Model/Payment/Product.php

```diff
@@ -52,6 +50,60 @@ private function getItemPrice($item, $productType)
         return $item->getPrice();
     }
 
+    /**
+     * @param array $list
+     * @param $order
+     * @return array
+     */
+    private function buildTax(array $list, $order)
+    {
+        if ($order->getTaxAmount() > 0) {
+            $productId = $this->findOrCreateProduct('taxa', 'taxa');
```

Comment:
O segundo parâmetro é exibido no painel, desse modo faz sentido deixar um nome mais humanizado :smiley: 
```suggestion
            $productId = $this->findOrCreateProduct('taxa', 'Taxa');
```